support/ebpf/dotnet_tracer.ebpf.c

```diff
@@ -10,30 +10,30 @@
 #include "types.h"
 
 // The number of dotnet frames to unwind per frame-unwinding eBPF program.
-#define DOTNET_FRAMES_PER_PROGRAM   5
+#define DOTNET_FRAMES_PER_PROGRAM 5
 
 // The maximum dotnet frame length used in heuristic to validate FP
-#define DOTNET_MAX_FRAME_LENGTH     8192
+#define DOTNET_MAX_FRAME_LENGTH 8192
 
 // Keep in sync with dotnet interpreter code
-#define DOTNET_CODE_JIT       0x1f
+#define DOTNET_CODE_JIT 0x1f
 #define DOTNET_CODE_FLAG_LEAF 0x80
 
 // Map from dotnet process IDs to a structure containing addresses of variables
 // we require in order to build the stack trace
 bpf_map_def SEC("maps") dotnet_procs = {
-  .type = BPF_MAP_TYPE_HASH,
-  .key_size = sizeof(pid_t),
-  .value_size = sizeof(DotnetProcInfo),
-  .max_entries = 1024,
+    .type = BPF_MAP_TYPE_HASH,
+    .key_size = sizeof(pid_t),
+    .value_size = sizeof(DotnetProcInfo),
+    .max_entries = 1024,
```

Comment:
In general I see an indentation of 2 spaces. Can you adjust the rules to have 2 spaces here as well?

Reply:
should be resolved with https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/310/commits/bf6f40cb141025cd3f18a3ca1677388275eff673